lang/rust/rust-values.mk

```diff
@@ -80,6 +80,8 @@ ifneq ($(CONFIG_RUST_SCCACHE),)
 	SCCACHE_DIR=$(RUST_SCCACHE_DIR)
 endif
 
+CARGO_RUSTFLAGS+=-Clink-arg=-fuse-ld=$(TARGET_LINKER)
```

Comment:
Please move it below `CARGO_PKG_PROFILE:=$(if $(CONFIG_DEBUG),dev,release)`, i.e. at last.